apps/web/app/future/routing-forms/page.tsx

```diff
@@ -1,30 +1,7 @@
-import type { PageProps } from "app/_types";
-import { type GetServerSidePropsContext } from "next";
-import { cookies, headers } from "next/headers";
 import { redirect } from "next/navigation";
-import z from "zod";
 
-import { buildLegacyCtx } from "@lib/buildLegacyCtx";
-
-const paramsSchema = z
-  .object({
-    pages: z.array(z.string()),
-  })
-  .catch({
-    pages: [],
-  });
-
-const getPageProps = async (context: GetServerSidePropsContext) => {
-  const { pages } = paramsSchema.parse(context.params);
-
-  return redirect(`/apps/routing-forms/${pages.length ? pages.join("/") : ""}`);
-};
-
-const Page = async ({ params, searchParams }: PageProps) => {
-  const legacyCtx = buildLegacyCtx(headers(), cookies(), params, searchParams);
-  await getPageProps(legacyCtx);
-
-  return null;
+const Page = () => {
+  redirect("/apps/routing-forms/forms");
```

Comment:
Heads up redirecting to this future route throws a client side error.

Reply:
Thanks, Joe, I pushed a fix!